Harvest.Jobs.Invoice/Program.cs

```diff
@@ -26,9 +27,39 @@ static void Main(string[] args)
             // setup di
             var provider = ConfigureServices();
 
+            
+

```

Comment:
How about instead of doing all the GetAwaiter() stuff you just make an async method which handles the work and then call it from Main().  i think it would clean things up and also be more flexible in case more logic needed to be added to main later.